src/main/java/frc/robot/commands/CombinedIntakeCDSForwardCommand.java

```diff
@@ -5,22 +5,29 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
 import frc.robot.subsystems.CDSSubsystem;
 import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
 
 public class CombinedIntakeCDSForwardCommand extends CommandBase {
   /** Creates a new OuttakeCommand. */
   private final CDSSubsystem CDSSubsystem;
 
+  private final ShooterSubsystem shooterSubsystem;
   private final IntakeSubsystem intakeSubsystem;
 
   public CombinedIntakeCDSForwardCommand(
-      IntakeSubsystem mIntakeSubsystem, CDSSubsystem mCDSSubsystem) {
+      IntakeSubsystem mIntakeSubsystem,
+      CDSSubsystem mCDSSubsystem,
+      ShooterSubsystem mShooterSubsystem) {
     // Use addRequirements() here to declare subsystem dependencies.
     addRequirements(mIntakeSubsystem);
     addRequirements(mCDSSubsystem);
+    addRequirements(mShooterSubsystem);
```

Comment:
To test: make sure adding Shooter subsystem as a requirement doesn't prevent us from using the shoot commands when we expect to. Should all work seamlessly together.